dev-client/src/components/home/SiteMap.tsx

```diff
@@ -161,7 +161,11 @@ const SiteMap = (
 
   const onPress = useCallback(
     async (feature: GeoJSON.Feature) => {
-      if (feature.geometry !== null && feature.geometry.type === 'Point') {
+      if (
+        calloutState.kind === 'none' &&
+        feature.geometry !== null &&
```

Comment:
question: do we want to check for not null or for truthiness? Can we use `feature.geometry?.type`?